hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/SchedulingPolicy.java

```diff
@@ -92,7 +92,7 @@ public static SchedulingPolicy parse(String policy)
     return getInstance(clazz);
   }
   
-  public void initialize(Resource clusterCapacity) {}
+  public void initialize(FSContext fsContext) {}
```

Comment:
Since this method is in an @Public class, let us add a new method and deprecate the old method.

Let us also add javadoc for both methods.